lib/cli.js

```diff
@@ -280,7 +282,14 @@ exports.run = async () => { // eslint-disable-line complexity
 		});
 		watcher.observeStdin(process.stdin);
 	} else {
-		const runStatus = await api.run(files);
+		const {filenames, ranges, ignored} = rangeParser.parseFileSelections(files);
+		if (ignored.length > 0) {
+			for (const file of ignored) {
+				console.error(`Ignoring other tests from ${file} since you selected lines.`);
```

Comment:
We can't write to the console from here, it messes with the reporters. I think for now we can use `exit()` here and explain that if you select lines, you shouldn't then also specify the same file without a line selection, and then print the offending files.